### ISSUE TITLE
figures sharpness

### ISSUE THREAD

**Issue by wangjiawen2013** [scverse/spatialdata-plot#317]
Hi,
The figures plotted by spatialdata is not as clear as that of squidpy (perhaps because the scatter point size is too small, but I didn't find where to adjust the point size), and spatialdata_plot is a lot more slowly than squidpy when plotting. how to improve it ?
Here's figures from spatialdata tutorial: https://spatialdata.scverse.org/en/latest/tutorials/notebooks/notebooks/examples/squidpy_integration.html
![image](https://github.com/user-attachments/assets/e5d95ed4-bcc7-4ec7-8d66-b2e1c3e822c7)

**Comment by wangjiawen2013** [comment]
@LucaMarconato 
I have tried the latest spatialdata_plot, now the default method is `datashader`. Here is the figure generated by `datashader`:
![image](https://github.com/user-attachments/assets/99559eb6-e2a1-49ac-b806-9a9855566e90)
And this is generated by `matplotlib`:
![image](https://github.com/user-attachments/assets/2a500f71-96b9-4c8c-bf8f-f2ad80a55731)
It is expect to be the same except the running time, but as you can see the two figures differs a lot !  what is more strange is that the running time of `datashader` is 180s, comparing with 105s of `matplotlib`. It seems that `datashader` is even more slowly than `matplotlib`. There are 250,000 cells in my dataset.

**Comment by wangjiawen2013** [comment]
Then I changed the region from "cell_circles" to "cell_boundaries", the figures looks clearly. Is it better to plot "cell_boundaries" instead of "cell_circles" ?
```
sdata["table"].obs["region"] = "cell_boundaries"
sdata.set_table_annotates_spatialelement("table", region="cell_boundaries")
sdata.pl.render_shapes("cell_boundaries", color="n_counts").pl.show()
```
![image](https://github.com/user-attachments/assets/17ab4f82-c3a7-45b4-bfbb-417aa3060677)

**Comment by wangjiawen2013** [comment]
Another little issue:  Now there are no x_axis and y_axis labels on the figures.
![image](https://github.com/user-attachments/assets/f1c7d72b-8edb-4afe-9367-a0b6a862aadb)

**Comment by LucaMarconato** [comment]
Thanks for the details.

- The performance problem of datashader is being addressed here: https://github.com/scverse/spatialdata-plot/pull/309. Can you try it you please?
- The difference in plot is being investigated here: https://github.com/scverse/spatialdata-plot/issues/311, please follow the discussion there for updates.

> Another little issue: Now there are no x_axis and y_axis labels on the figures.

I could not reproduce, please open a second issue for tracking this other problem and add some details in which code you used. In the plots I made the ticks are rendered correctly.

**Comment by LucaMarconato** [comment]
> Is it better to plot "cell_boundaries" instead of "cell_circles" ?

Yes, it is better to plot the `cell_boundaries` as `cell_circles` are approximations of the cells. Nevertheless, plotting the second is generally faster, so for quick glances at the data is probably preferred.

**Comment by wangjiawen2013** [comment]
> Thanks for the details.
> 
> * The performance problem of datashader is being addressed here: [speed up datashader by using canvas size equal to image size #309](https://github.com/scverse/spatialdata-plot/pull/309). Can you try it you please?
> * The difference in plot is being investigated here: [Wrong colors when using `method='datashader'` #311](https://github.com/scverse/spatialdata-plot/issues/311), please follow the discussion there for updates.
> 
> > Another little issue: Now there are no x_axis and y_axis labels on the figures.
> 
> I could not reproduce, please open a second issue for tracking this other problem and add some details in which code you used. In the plots I made the ticks are rendered correctly.

I don't know which plot is the correct one, could you tell me the `datashader` or `matplotlib` should be used now ? Or the issues will be fixed next release ? If so, I well wait to the next release.

**Comment by LucaMarconato** [comment]
We are trying to fix the issue by the next release. I would suggest to use `matplotlib` until then. CC @timtreis
